### ISSUE TITLE
Octal rule is broken

### ISSUE THREAD

**Issue by JordanP** [haypo/sixer#5]
Sixer basically wants to replace every occurrence of "0[0-9]" by "0o[0-9]" even if the zero is in the middle of a string.

```
# The following lines define string.
uuid1 = "994610d6-0476-4018-a644-a2602ef5d4aa"
mac_address = "00:16:3e:33:44:55"
```

On that sample file, sixer changes both `uuid1` and `mac_address` but these variables are not Octal int.

**Comment by JordanP** [comment]
Fixed by https://github.com/haypo/sixer/commit/658b77a09cb8ef961f932b122b9cbf248a40bec6

**Comment by vstinner** [comment]
Fixed in sixer 1.5.
